src/app/views/censusdata/create3.html

```diff
@@ -0,0 +1,49 @@
+<div class="container">
+    <div ng-include="'/app/views/censusdata/progressBar.html'"></div>
+    {{censusForm.step = 3}}
+    <form class="form-horizontal" role="form"> 
+        <div class="form-group panel panel-primary">
+            <div class="panel-heading">Disturbance<span style="float:right;">Report violations to 1-800-792-4263</span></div>
+            <div class="panel-body">                          
+            
+            <div class="form-group">        
+                <div class="col-md-12">
+                    <!--<div>{{ 6/2 }}</div>-->
+                    <table class="table table-striped">
+                        <thead>
+                            <tr class="info">
+                                <th>Types</th>
+                                <th>Quantity</th>
+                                <th>Duration (nearest 5 minutes)</th>
+                                <th>Result or Affect (observed behavior)</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr ng-repeat="(i, disturbance) in data.disturbanceTypeInfo track by $index ">
```

Comment:
Same approach is taken as described above. So the json object already have disturbanceId and birdId which will be passed to the API.